dev/dependencies.sh

```diff
@@ -39,8 +39,10 @@ DEP=""
 function mvn_build_classpath() {
   $MVN -P$MODULE clean install -DskipTests -am -pl $MVN_MODULES
   $MVN -P$MODULE dependency:build-classpath -am -pl $MVN_MODULES | \
-    grep -v "INFO\|WARN" | \
-    # This will skip the first two lines 
+    grep -A1 "Dependencies classpath:" | \
+    grep -v "^--$" | \
+    grep -v "Dependencies classpath:" | \
+    # This will skip the first two lines
     tail -n +3 | \
```

Comment:
after we used `grep -A1`, we don't need this skip now?

Reply:
Looks like it can be removed.